src/org/camunda/latera/bss/connectors/hoper/Hoper.groovy

```diff
@@ -18,12 +18,11 @@ class Hoper {
     def ENV     = System.getenv()
 
     this.url      =  execution.getVariable("hoperUrl")      ?: ENV['HOPER_URL']     ?: 'http://hoper:3000'
-    this.version  = (execution.getVariable("hoperVersion")  ?: ENV['HOPER_VERSION'] ?: 2)?.toInteger()
     this.user     =  execution.getVariable("hydraUser")     ?: ENV['HYDRA_USER']
     this.password =  execution.getVariable("hydraPassword") ?: ENV['HYDRA_PASSWORD']
 
     this.http = new HTTPRestProcessor(
-      baseUrl   : "${this.url}/rest/v${this.version}",
```

Comment:
And what are you suppose to do then v3 of REST API will come?

Reply:
Change it.
But I think it should be in `hoperUrl`, because you need only rest url from hoper.
And you don't use it anywhere else.

Reply:
You can't rely on just hoper url because entities names are different for different REST API versions, like company -> organization. Using separated version field is better than parsing URL with version inside.

Reply:
After discussion was decided that Hydra and REST API versions in camunda-ext will be fixed.